airflow/contrib/hooks/gcp_spanner_hook.py

```diff
@@ -55,31 +54,36 @@ def get_client(self, project_id):
             self._client = Client(project=project_id, credentials=self._get_credentials())
         return self._client
 
+    @GoogleCloudBaseHook.fallback_to_default_project_id
     def get_instance(self, project_id, instance_id):
-        # type: (str, str) -> Optional[Instance]
+        # noinspection LongLine
         """
         Gets information about a particular instance.
 
-        :param project_id: The ID of the project which owns the Cloud Spanner Database.
+        :param project_id: The ID of the  GCP project that owns the Cloud Spanner
+            database. if set to None, default project_id is used from service account.
         :type project_id: str
         :param instance_id: The ID of the Cloud Spanner instance.
         :type instance_id: str
         :return: Representation of a Cloud Spanner Instance. See:
             https://googleapis.github.io/google-cloud-python/latest/spanner/instance-api.html#google.cloud.spanner_v1.instance.Instance
         :rtype: object
         """
+        # noinspection PyUnresolvedReferences
```

Comment:
Please remove all comments that disable style check. As we had a talk earlier that can be a separate PR where we resolve that issue

Reply:
Done that. The PR for abstract class is in the making :)